net: shell: keep event_mon_handler compatible to k_thread_entry_t

laroche opened the pull request:
Nearly all other code places for k_thread_entry_t also keep unused params in place to stay compatibel with k_thread_entry_t.

rlubos reviewed (approved):
Please fix the typo in the commit title (`compatibel`), looks good otherwise.

laroche committed:
net: shell: keep event_mon_handler compatible to k_thread_entry_t

Nearly all other code places for k_thread_entry_t also keep
unused params in place to stay compatible with k_thread_entry_t.

Signed-off-by: Florian La Roche <Florian.LaRoche@gmail.com>

jukkar commented:
Note that you have to edit your local commit and fix the message, and then force push to github. Editing the PR title does is not enough.

laroche commented:
Seems the change from "compatibel -> compatible" took some time to go through at github.
Should all be correct now.
